README.md

```diff
@@ -1,4 +1,147 @@
 # Tribute-Page
+ <h3><b>Microverse README Template</b></h3>
 
+</div>
 
-lpovde
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [A Tribute Page] <a name=a tribute page></a>
+
+
+**[A Tribute Page]** is a simple project built with html and css describing a very good man that ever lived.
+
+## 🛠 Built With
+1. Html
+2. Css
+3. Linters
```

Comment:
I think it would be better to include linters in your code as mentioned here. Linters makes your code better and easy for other programmers.

Reply:
- [ ] Good job using the inline comments and providing a reason for your requested changes 👏, but please make sure to be more descriptive and specific in your comments as the document you've highlighted seems to have the text "Linters" or maybe you meant the linters check. You can see how confusing😵 this becomes so please try to fix it.